### PR TITLE
Chimney A/B Test

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
An A/B test to see if adding a chimney to the home icon increases clickthrough rate.

screen shot 2015-03-09 at 17 01 16

cute-collapsing-towers-animation

### CODE REVIEW

**Comment by rich-nguyen** [comment]
:+1:

**Comment by crifmulholland** [comment]
This is a game-changer

File: static/src/javascripts/projects/common/modules/experiments/tests/chimney.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        this.id = 'Chimney';
+        this.start = '2015-03-08';
+        // far future expiration, only really using the test to bucket users, which we can use for targeting in dfp
+        this.expiry = '2015-03-16';
+        this.author = 'Sam Morris';
+        this.description = 'Testing to see what affect a chimney on the home icon has.';
+        this.audience = 0.1;
+        this.audienceOffset = 0.43;
+        this.successMeasure = 'A chimney increases the likelihood of someone clicking on the home icon.';
+        this.audienceCriteria = 'All users';
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/experiments/tests/chimney.js]: mobile users only!

**Comment by phamann** [comment]
:laughing:

**Comment by paperboyo** [comment]
cat

**Comment by mattandrews** [comment]
What was the outcome of this?

**Comment by crifmulholland** [comment]
Sam can run an a/b test.

And nobody cares about chimneys.

**Comment by mattandrews** [comment]
What if there was some animated smoke coming out of it?

**Comment by sammorrisdesign** [comment]
If the test was pro-chimney I would have spent the time on adding smoke.

**Pull request merged**